notebook/static/tree/js/fileupload.js

```diff
@@ -0,0 +1,1462 @@
+/*
```

Comment:
External dependencies should not be included in this repo. If it is needed, it should be added as a dependency in `bower.json` and loaded with `require`.